thunder/torch/__init__.py

```diff
@@ -2687,6 +2687,19 @@ def clone(a: TensorProxy, *, memory_format=torch.preserve_format) -> TensorProxy
 register_method("clone", clone)
 
 
+@torchsymbol(torch.nn.functional.glu, is_method=False)
+def glu(a: TensorProxy, /, dim: int = -1):
+    dim = -1 if dim is None else dim
```

Comment:
When would `dim` be `None`? The default is -1 so I wasn't sure at glance.
It might be better to make sure `0 <= dim < a.ndim` or `dim == -1`